Code/GraphMol/Canon.cpp

```diff
@@ -18,7 +18,7 @@
 
 namespace RDKit {
 namespace Canon {
-namespace {
+namespace details {
```

Comment:
is there a reason to not use an anonymous namespace?  these detail functions don't appear to be used outside the file.

Reply:
if I leave them in the anonymous namespace then they can't be tested directly and it was much easier to test these directly than to construct test molecules which test them indirectly.

Reply:
good enough for me.